Tests/PostgresNIOTests/PostgresNIOPerformanceTests.swift

```diff
@@ -0,0 +1,302 @@
+import Logging
+import PostgresNIO
+import XCTest
+import NIOTestUtils
+
+final class PostgresNIOPerformanceTests: XCTestCase {
```

Comment:
Should we call this just `PerformanceTests` since its already in the `PostgresNIOTests` module?

Reply:
Admittedly a matter of indifference to me, so done 🙂